lib/s3mpi/converters.rb

```diff
@@ -1,6 +1,21 @@
 require 's3mpi/converters/csv'
+require 's3mpi/converters/json'
+require 's3mpi/converters/identity'
 
 module S3MPI
   module Converters
+
+    class UnknownConverterError < StandardError; end
+
+    def converter(as)
+      case as
+      when :json then Converters::JSON
+      when :csv  then Converters::CSV
+      when :string, :identity then Converters::Identity
+      else
+        raise UnknownConverterError, "#{as.inspect} is not a known converter!"
```

Comment:
it's harder to find / notice generic errors in reporting tools like Honeybadger or NewRelic